docs/wordpress.md

```diff
@@ -13,85 +13,126 @@ weight=6
-Dockerfiles, see the
-[Docker user guide](https://docs.docker.com/engine/userguide/dockerimages/#building-an-image-from-a-dockerfile) and the
-[Dockerfile reference](https://docs.docker.com/engine/reference/builder/). In
-this case, your Dockerfile should be:
-
-    FROM orchardup/php5
-    ADD . /code
-
-This tells Docker how to build an image defining a container that contains PHP
-and WordPress.
-
-Next you'll create a `docker-compose.yml` file that will start your web service
-and a separate MySQL instance:
-
-    version: '2'
-    services:
-      web:
-        build: .
-        command: php -S 0.0.0.0:8000 -t /code
-        ports:
-          - "8000:8000"
-        depends_on:
-          - db
-        volumes:
-          - .:/code
-      db:
-        image: orchardup/mysql
-        environment:
-          MYSQL_DATABASE: wordpress
-
-A supporting file is needed to get this working. `wp-config.php` is
-the standard WordPress config file with a single change to point the database
-configuration at the `db` container:
-
-    <?php
-    define('DB_NAME', 'wordpress');
-    define('DB_USER', 'root');
-    define('DB_PASSWORD', '');
-    define('DB_HOST', "db:3306");
-    define('DB_CHARSET', 'utf8');
-    define('DB_COLLATE', '');
-
-    define('AUTH_KEY',         'put your unique phrase here');
-    define('SECURE_AUTH_KEY',  'put your unique phrase here');
-    define('LOGGED_IN_KEY',    'put your unique phrase here');
-    define('NONCE_KEY',        'put your unique phrase here');
-    define('AUTH_SALT',        'put your unique phrase here');
-    define('SECURE_AUTH_SALT', 'put your unique phrase here');
-    define('LOGGED_IN_SALT',   'put your unique phrase here');
-    define('NONCE_SALT',       'put your unique phrase here');
-
-    $table_prefix  = 'wp_';
-    define('WPLANG', '');
-    define('WP_DEBUG', false);
-
-    if ( !defined('ABSPATH') )
-        define('ABSPATH', dirname(__FILE__) . '/');
-
-    require_once(ABSPATH . 'wp-settings.php');
+1. Create an empty project directory.
+
+    You can name the directory something easy for you to remember. This directory is the context for your application image. The directory should only contain resources to build that image.
+
+    This project directory will contain a `Dockerfile`, a `docker-compose.yaml` file, along with a downloaded `wordpress` directory and a custom `wp-config.php`, all of which you will create in the following steps.
+
+2. Change directories into your project directory.
+
+    For example, if you named your directory `my_wordpress`:
+
+        $ cd my-wordpress/
+
+3. Create a `Dockerfile`, a file that defines the environment your app is going to run in. For more information on how to write Dockerfiles, see the [Docker user guide](https://docs.docker.com/engine/userguide/dockerimages/#building-an-image-from-a-dockerfile) and the [Dockerfile reference](https://docs.docker.com/engine/reference/builder/). In this case, your Dockerfile should include these two lines:
```

Comment:
Docker Engine user guide  -> trying to specific and not generalize everything under a Docker label.

"Docker container" is a Docker resources.